### ISSUE TITLE
error: declaration shadows a type alias

### ISSUE THREAD

**Issue by Finkman** [fmtlib/fmt#1933]
I use fmt in-source and included `#include <fmt/format.h>`

Then the clang 10.0.1 compiler tells:

```
[build] /home/fins/Development/EclipseWorkspace/wipoframework/thirdparty/fmt/include/fmt/format.h:1601:11: error: declaration shadows a type alias in 'int_writer<OutputIt, Char, UInt>' [-Werror,-Wshadow]
[build]     using iterator = remove_reference_t<decltype(reserve(out, 0))>;
[build]           ^
[build] /home/fins/Development/EclipseWorkspace/wipoframework/thirdparty/fmt/include/fmt/format.h:1490:9: note: previous declaration is here
[build]   using iterator =
[build]         ^
```

**Comment by vitaut** [comment]
What library version?

**Comment by Finkman** [comment]
7.0.3

**Comment by vitaut** [comment]
This has already been fixed in #1808.

**Comment by Finkman** [comment]
sorry for the noise
